partner_firstname/models/partner.py

```diff
@@ -145,3 +188,10 @@ def _install_partner_firstname(self):
         # Force calculations there
         records._inverse_name()
         _logger.info("%d partners updated installing module.", len(records))
+
+    # Python constraint gives a more explicit error
+    _sql_constraints = [(
+        'check_name',
+        "CHECK( 1=1 )",
```

Comment:
This is the same thing you did in your portage.

I tried to keep the sql constraint but it was redundant